### DISCUSSION TITLE
Add package to build from a dependency?

### DISCUSSION THREAD

**Discussion by rlaphoenix** [python-poetry/poetry#6796] [Q&A]
I have a namespace named 'foo'. Under it there's two sub-packages. 'core' and 'commands'. When I run `poetry build` it works great and I end up with both sub-packages. However, I have a second project named 'foo-repository' added as a dependency. It's structure has the same named 'foo' namespace folder with two packages, 'core' (again) and 'services'.

When I run `poetry build` I was expecting to have all three folders in the whl. Is this possible?

I have poetry recognizing the namespace in `foo-repository` with:

```toml
packages = [
    { include = "foo" }
]
```

Under poetry I can recognize the paths are different for these two namespaces. The 'core' and 'commands' when imported are in the main project git repository location. The 'services' and 2nd 'core' from 'foo-repository' are importing from the main project's venv.

**Comment by neersighted** [comment]
What you are looking for is not how Python packaging works normally -- you need to build both packages and install them both (the namespaces will not conflict if you use implicit namespace paths -- if `poetry install` works, both packages will be installable on their own).

If you are interested in vendoring the third module in so that there is only one package, that feature is tracked at #4583.

**Reply by rlaphoenix** [reply]
Yes this is correct. They are both installable separately. But yeah vendoring them is what I was hoping to do. Thanks for the link.
